### ISSUE TITLE
Disabling Currency and Permanent Alt View

### ISSUE THREAD

**Issue by Kanegasi** [SavedInstances/SavedInstances#141]
I suggested this once on wowace/curseforge tickets with the old author, but they simply rejected and closed the ticket without any reply. I'd link the ticket, but it seems you guys recently disabled those tickets to link here. Hopefully you guys are more reasonable.

These two suggestions are sort of related. I would like an option to disable all currency tracking and an option for the broker tooltip to act like I have alt held down all the time. I like seeing all my lockouts, expired or not, but holding alt brings up currencies even if I have them disabled. I have another method of tracking currency, but I really like this addon's presentation of lockouts and other daily/weekly tracking.

For many many months, possibly years because I can't remember the original ticket date, I've been editing the addon myself every single update because both spots that do what I want are declared locally so my personal addon can't change them.

To see what I'm aiming for, edit the current version (8.0.3) with these three changes and see the tooltip ingame (these three changes are what I've been doing):

```
line 123: local currency = {} --[[

line 158: }]]

line 3767:   return true -- (IsAltKeyDown() and true) or false
```

**Comment by LiangYuxuan** [comment]
I'm not quite sure but it seems you want a new view which showing all lockouts and no currencies.

**Comment by Kanegasi** [comment]
Yes. I want the "hold alt" view to be permanent with no currencies shown. I still perform the changes each addon release, here's what they look like in version 8.0.8: https://imgur.com/a/IIgBm42
